modules/mobile-agents/android/client/client/src/main/java/org/wso2/emm/agent/RegistrationActivity.java

```diff
@@ -244,4 +248,28 @@ private void loadAuthenticationErrorActivity() {
 		startActivity(intent);
 	}
 
+	/**
+	 *  This method is used to invoke getEffectivePolicy in the backend
+	 */
+	private void getEffectivePolicy() {
```

Comment:
Yes, it returns the policy. But in this method it makes a network call and the result will be handled by callback handler.